class NotFoundError(Exception):
    pass

class MultipleFoundError(Exception):
    pass
